model/patch_lifecycle.go

```diff
@@ -142,17 +142,17 @@ func ConfigurePatch(ctx context.Context, settings *evergreen.Settings, p *patch.
 			return http.StatusInternalServerError, errors.Wrap(err, "setting patch parameters")
 		}
 	}
-
 	// update the description for both reconfigured and new patches
 	if err = p.SetDescription(patchUpdateReq.Description); err != nil {
 		return http.StatusInternalServerError, errors.Wrap(err, "setting description")
 	}
 
-	// update the description for both reconfigured and new patches
-	if err = p.SetVariantsTasks(tasks.TVPairsToVariantTasks()); err != nil {
-		return http.StatusInternalServerError, errors.Wrap(err, "setting description")
+	patchVariantTasks := tasks.TVPairsToVariantTasks()
+	if len(patchVariantTasks) > 0 {
+		if err = p.SetVariantsTasks(patchVariantTasks); err != nil {
+			return http.StatusInternalServerError, errors.Wrap(err, "setting description")
+		}
 	}
-	p.Activated = true
```

Comment:
If this isn't set here anymore, when does it get set? In `addNewTasksAndBuildsForPatch`?

Also for my own understanding, does a patch being activated mean that it's finalized (i.e. tasks will be scheduled)? Or does it mean something else?

Reply:
Feel like the best part of that doc comment is that I wrote it and apparently forgot.